labelCloud/io/labels/kitti.py

```diff
@@ -32,10 +35,14 @@ def import_labels(self, pcd_path: Path) -> List[BBox]:
                 line_elements = line.split()
                 centroid = tuple([float(v) for v in line_elements[11:14]])
                 if self.transformed:
-                    centroid = centroid[2], -centroid[0], centroid[1] - 2.3
+                    T_c2l = self.calc_cam2lidar(calib_path)
+                    xyz1 = np.insert(np.asarray(centroid), 3, values=[1])
+                    xyz1 = T_c2l @ xyz1
+                    centroid = tuple([float(n) for n in xyz1[:-1]])
                 dimensions = tuple([float(v) for v in line_elements[8:11]])
```

Comment:
I think the if condition can be merged if the `dimensions` a specified before the first if condition.

Reply:
Changed that myself.